{"uuid":"08680a4b-0cb4-48f9-930f-9e62188839ca","historyId":"4b0a14f3251ae56b8adec22c08aa6b1f","fullName":"pageTestEvents.RegisterPageEvents.testUserRegistration","labels":[{"name":"package","value":"pageTestEvents.RegisterPageEvents"},{"name":"testClass","value":"pageTestEvents.RegisterPageEvents"},{"name":"testMethod","value":"testUserRegistration"},{"name":"parentSuite","value":"Suite_Parallel"},{"name":"suite","value":"Test Parallel SignIn and Register modules.Browser:Edge "},{"name":"subSuite","value":"pageTestEvents.RegisterPageEvents"},{"name":"host","value":"Anvi"},{"name":"thread","value":"24648@Anvi.TestNG-tests-2(43)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"testUserRegistration","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [user already exists] but found [password_mismatch:The two password fields didn’t match.]","trace":"java.lang.AssertionError: expected [user already exists] but found [password_mismatch:The two password fields didn’t match.]\r\n\tat org.testng.Assert.fail(Assert.java:111)\r\n\tat org.testng.Assert.failNotEquals(Assert.java:1578)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:150)\r\n\tat org.testng.Assert.assertEquals(Assert.java:132)\r\n\tat org.testng.Assert.assertEquals(Assert.java:656)\r\n\tat org.testng.Assert.assertEquals(Assert.java:666)\r\n\tat pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)\r\n\tat pageTestEvents.RegisterPageEvents.testUserRegistration(RegisterPageEvents.java:68)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:578)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:739)\r\n\tat org.testng.TestRunner.run(TestRunner.java:614)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)\r\n\tat org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)\r\n\tat java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1589)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"arg3","value":"user already exists"},{"name":"arg2","value":"Numpysdet176"},{"name":"arg1","value":"Numpysdet176"},{"name":"browser","value":"edge"},{"name":"arg0","value":"Scrutinizers"}],"start":1731802858708,"stop":1731802862478}